providers/prefab/README.md

```diff
@@ -0,0 +1,56 @@
+# Unofficial Prefab OpenFeature Provider for GO
```

Comment:
hi @semanticart @jkebinger,

you are welcome to review.
this PR is similar to [Java provider PR](https://github.com/open-feature/java-sdk-contrib/pull/915).

Reply:
Hi @semanticart
Will you be able to review this from Prefab point of view?

Reply:
@liran2000 yes, I can take a look soon. Thanks!

Reply:
thanks.
@Kavindu-Dodan / @toddbaert you are welcome to review too.